ggshield/core/git_shell.py

```diff
@@ -319,3 +346,10 @@ def tar_from_ref_and_filepaths(
             tar.addfile(tarinfo, fileobj=data)
 
     return tar_stream.getvalue()
+
+
+def get_empty_tar() -> bytes:
```

Comment:
`tar_from_ref_and_filepaths()` in `git_shell`, it makes less sense to me to have a `get_empty_tar()` function in this module.

I suggest either moving `get_empty_tar()` to `core.file_utils` or create a `core.tar_utils` module and move both `get_empty_tar()` and `tar_from_ref_and_filepaths()` to it.